### DISCUSSION TITLE
is there option to restore the directory on restart?

### DISCUSSION THREAD

**Discussion by zlev-intel** [chrisant996/clink#616] [General]
for example, I'm in directory c:\sources\data\my_app closing the cmd; can I get it in a same directory after restart?
The same with multiple CMD windows in Win11?

**Comment by chrisant996** [comment]
The [cwdhistory.lua](https://github.com/chrisant996/clink-gizmos/blob/8e49a4c0494c3ebd1c8f6707795930ba00f51248/cwdhistory.lua#L61) script in the [clink-gizmos](https://github.com/chrisant996/clink-gizmos) repo has a setting that can do that (and also many other features).
